### PR TITLE
feature Adding index.d.ts for typescript usage

### PR DESCRIPTION
**Pull request by airon-assustadus**
Adding index.d.ts for typescript usage

Fixes #
Adding index.d.ts for typescript usage

## Proposed Changes

  - Adding index.d.ts for typescript usage
  -
  -

### CODE REVIEW

**Comment by aitoraznar** [comment]
Would you mind to you take a look? This looks very useful @iuccio.

**Comment by iuccio** [comment]
Hi @airon-assustadus can you add please some documentation about the TS feature and some tests for the fixing header?

File: index.d.ts
Comment (iuccio) [inline index.d.ts:1]: Hi @airon-assustadus can you add please some documentation about the TS feature and some tests for the fixing header?

**Review by iuccio — COMMENTED** [review]
Hi @airon-assustadus can you add please some documentation about the TS feature and some tests for the fixing header?

File: src/csvToJson.js [hunk 1]
```diff
@@ -69,8 +69,14 @@ class CsvToJson {
     let lines = parsedCsv.split(newLine);
     let fieldDelimiter = this.getFieldDelimiter();
     let index = this.getIndexHeader();
-    let headers = lines[index].split(fieldDelimiter);
-
+    let headers;
```
Comment (iuccio) [inline src/csvToJson.js:72]: Add some tests

**Review by iuccio — APPROVED** [review]

**Comment by BryanBerger98** [comment]
Hi @iuccio ! When this PR will be merged ?

**Comment by iuccio** [comment]
> Hi @iuccio ! When this PR will be merged ?

@BryanBerger98 I'll try this weekend. Do you think this feature needs some extra documentation?

**Comment by iuccio** [comment]
@BryanBerger98 feature already merged.

**Comment by BryanBerger98** [comment]
Sorry, I didn't see your comments. Thanks!

**Pull request merged**